cmd/homi/docker/service/validator.go

```diff
@@ -144,29 +144,25 @@ var validatorTemplate = `{{ .Name }}:
         --unlock "75a59b94889a05c03c66c3c84e9d2f8308ca4abd" \
         --password "/klaytn/password.txt" \
 {{- end}}
+{{- if eq .NodeType "scn" }}
```

Comment:
Should we include `spn/sen` for this?

Reply:
I think it is not necessary. Since only scconsensus options is for the SCN, the option is handled with .NodeType. {{- if .ParentChainId }} is used for SPN and SEN instead.

Reply:
You are right. I'll update the code.

Reply:
Fixed. PTAL.